recipe/meta.yaml

```diff
@@ -1,22 +1,23 @@
 {% set name = "lda" %}
-{% set version = "2.0.0" %}
+{% set version = "3.0.2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: d8d7c0bc42208902a9e34261b76793a84f59e6c69cfd7ca12ce982cdaf7c3020
+  sha256: 76fc6fbb066b6d1ec0360a1541c5e1c8b69a728666525e72644c4d5332fc778a
 
 build:
-  number: 1
+  number: 0
   script: {{ PYTHON }} -m pip install . -vv
   skip: true  # [py>=37]
```

Comment:
Is this `skip` right or is this meant to be reversed?

Appears CI is getting turned off because of it

In any event conda-forge is on [Python 3.8+]( https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/880b27bddf121255acc488447d3fbfe985d89265/recipe/conda_build_config.yaml#L771-L777 ). So this can likely be dropped (assuming newer Python versions work)

Would make sure to [re-render]( https://conda-forge.org/docs/maintainer/updating_pkgs/#rerendering-feedstocks ) after